pkg/auth/providers/saml/saml_client.go

```diff
@@ -139,7 +139,7 @@ func InitializeSamlServiceProvider(configToSet *v3.SamlConfig, name string) erro
 	sp.IDPMetadata.EntityID = idm.EntityID
 	sp.IDPMetadata.SPSSODescriptors = idm.SPSSODescriptors
 	sp.IDPMetadata.IDPSSODescriptors = idm.IDPSSODescriptors
-	if name == ADFSName {
+	if name == ADFSName || name == OkTAName {
```

Comment:
Is this done because we tested integration between an Okta Identity Provider and Rancher's ADFS provider?

Reply:
Year!